### PR TITLE
Refactor tests to PEST

### PR DESCRIPTION
**Pull request by alexmanase**

### CODE REVIEW

File: tests/ConfigurationValidatorTest.php [hunk 1]
```diff
@@ -1,87 +1,73 @@
...
-
-        $this->assertEquals(count($invalidItems), $exceptionCounter);
     }
 
-    /** @test */
-    public function it_throws_an_exception_for_an_invalid_view()
-    {
-        $exceptionCounter = 0;
-        $views = ['', 'feed::missing', null, 'feed::rss'];
+    assertEquals(count($invalidItems), $exceptionCounter);
```
Comment (freekmurze) [inline tests/ConfigurationValidatorTest.php:52]: Instead of the PHPUnit assertion, use Pest's expectation API here and everywhere assertions are made.

**Review by freekmurze — CHANGES_REQUESTED** [review]

File: tests/ConfigurationValidatorTest.php (hunk 1 above)
Comment (alexmanase) [reply tests/ConfigurationValidatorTest.php:52]: Sure.

File: tests/FeedItemTest.php [hunk 2]
```diff
@@ -1,31 +1,20 @@
...
-    public function it_can_be_created_without_errors()
-    {
-        FeedItem::create()
-            ->title('A title')
-            ->category('a category')
-            ->link('https://spatie.be')
-            ->authorName('an author')
-            ->authorEmail('author@test.test')
-            ->updated(now());
+test('it can be created without errors', function () {
```
Comment (freekmurze) [inline tests/FeedItemTest.php:10]: When a test name starts with "it", use Pests `it` function and omit "it" from the test name.

**Review by freekmurze — CHANGES_REQUESTED** [review]

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**